src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_help.py

```diff
@@ -15,4 +15,9 @@
 helps['iot device create'] = """
             type: command
             short-summary: Register a device in your Azure IoT Hub.
+"""
+
+helps['iot'] = """
+    type: group
+    short-summary: Connect, monitor, and control millions of IoT assets
```

Comment:
The marketing text isn't terribly accurate since it only has two commands... one to create a hub and one to essentially add devices to a hub.

---
In reply to: [79516407](https://github.com/Azure/azure-cli/pull/923#discussion_r79516407) [](ancestors = 79516407)